package/Dockerfile

```diff
@@ -1,5 +1,15 @@
-FROM ubuntu:18.04
+FROM ubuntu:18.04 as intermediate
+
+#Label the image for cleaning after build process
+LABEL stage=intermediate
+
+RUN apt-get update \
+    && apt-get install -y \
+        git
+
+RUN git clone https://github.com/aquasecurity/kube-bench.git
```

Comment:
Do we need the entire history? If not, can we update to add `--depth=1`?

Reply:
Also, can we lock down the release tag as well?

Reply:
Could we use the source from the release page https://github.com/aquasecurity/kube-bench/archive/v0.3.1.zip

Reply:
locked down to tag v0.3.1 and with --depth=1

Reply:
@paynejacob I think with tag and depth we should be good?